tests/test_coordinator.py

```diff
@@ -17,49 +27,86 @@ def __init__(self):
+        self._running.set()
 
     def get_sensors(self):
-        return SensorValues()
+        self._running.wait()
+        return SensorValues(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0)
 
     def get_control(self, control_setting_name: ValueName) -> ControlSetting:
+        self._running.wait()
         return self.control_setting[control_setting_name]
 
     def set_control(self, control_setting: ControlSetting):
+        self._running.wait()
         self.control_setting[control_setting.name] = control_setting
 
 
 def mock_get_control_module(sim_mode):
     return ControlModuleMock()
 
 
-@pytest.mark.parametrize("control_setting_name", [ValueName.PIP,
-                                                  ValueName.PIP_TIME,
-                                                  ValueName.PEEP,
-                                                  ValueName.BREATHS_PER_MINUTE,
-                                                  ValueName.INSPIRATION_TIME_SEC])
-                                                  
+@pytest.mark.parametrize("control_setting_name", values.controllable_values)
 @patch('vent.controller.control_module.get_control_module', mock_get_control_module, Mock())
-
-def test_coordinator(control_setting_name):
-
+def test_local_coordinator(control_setting_name):
     coordinator = get_coordinator(single_process=True, sim_mode=True)
+    coordinator.start()
+    while not coordinator.is_running():
+        pass
     t = time.time()
     v = random.randint(10, 100)
     v_min = v - 5
     v_max = v + 5
 
-    # TODO: add test for start/stop
+    # TODO: add test for stop
     # TODO: add test for test reference
 
     c = ControlSetting(name=control_setting_name, value=v, min_value=v_min, max_value=v_max, timestamp=t)
     coordinator.set_control(c)
 
-    # TODO: this should be tight
-    time.sleep(0.1)
+    c_read = coordinator.get_control(control_setting_name)
+    assert c_read.name == c.name
+    assert c_read.value == c.value
+    assert c_read.min_value == c.min_value
+    assert c_read.max_value == c.max_value
+    assert c_read.timestamp == c.timestamp
+
+
+@pytest.mark.parametrize("control_setting_name", values.controllable_values)
+@patch('vent.controller.control_module.get_control_module', mock_get_control_module, Mock())
+def test_remote_coordinator(control_setting_name):
+    # wait before
+    while not is_port_in_use(rpc.default_port):
+        time.sleep(1)
+    coordinator = get_coordinator(single_process=False, sim_mode=True)
+    #TODO need to wait for rpc client start?
+    time.sleep(1)
+    coordinator.start()
+    while not coordinator.is_running():
+        pass
+    t = time.time()
+    v = random.randint(10, 100)
+    v_min = v - 5
+    v_max = v + 5
+
+    # TODO: add test for stop
+    # TODO: add test for test reference
+    # TODO: test racing condition
+
+    c = ControlSetting(name=control_setting_name, value=v, min_value=v_min, max_value=v_max, timestamp=t)
+    coordinator.set_control(c)
```

Comment:
Please add tests for each type of remote call, such as get_sensors, get_alarms, etc.